docs/sources/operators-guide/tools/query-tee.md

```diff
@@ -35,17 +35,26 @@ chmod +x query-tee
 ## Configure the query-tee
 
 The query-tee requires the endpoints of the backend Grafana Mimir clusters.
-You can configure the backend endpoints by setting the `-backend.endpoints` flag to a comma-separated list of HTTP or HTTPS URLs.
+You can configure the backend endpoints by setting the `-backend.endpoints` flag to a comma-separated list endpoints:
+
+- HTTP endpoints: via HTTP or HTTPS URLs. Example: `http://query-frontend:80`.
+- gRPC endpoints: via gRPC URI scheme. Example: `dns:///query-frontend:9095`.
```

Comment:
I don't think this is correct. We only support HTTP for the backend. In this PR we just added gRPC support as "input" protocol.

```suggestion
You can configure the backend endpoints by setting the `-backend.endpoints` flag to a comma-separated list of HTTP or HTTPS URLs.
```